client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -67,16 +66,23 @@ public void getRPT_caching() {
         verify(tokenCache, times(1)).update(any());
     }
 
-    @Test(expected = KeycloakAuthenticationException.class)
+    @Test
     public void getRPT_error_authn() {
```

Comment:
Could we rename this to something more meaningful? The purpose of the test is not immediately apparent to me (from the name itself).

Reply:
I renamed all the tests to something more meaningful.